.github/workflows/tests.yaml

```diff
@@ -74,3 +74,29 @@ jobs:
         MZ_CLUSTER: default
         MZ_SCHEMA: dbt_action_ci_cd
         DBT_PROJECT_FOLDER: 'tests/custom_profiles_test'
+    - name: dbt version (poetry)
+      uses: bobbyiliev/materialize-dbt-action@main
```

Comment:
```suggestion
      uses: MaterializeInc/dbt-action@main
```

Reply:
Fixed these. Though seeing the tests will pass on the branch as long as the main branch is passing